modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -72,6 +72,22 @@ This section provides the necessary details that enable you to control egress tr
 |`.q1w2.quay.rhcloud.com`
 |443
 |Provides core container images as a fallback when quay.io is not available.
+
+|`www.okd.io`
+|443
+|The `openshift.org` site redirects through `www.okd.io`.
+
+|`www.redhat.com`
+|443, 80
+|The `sso.redhat.com` site redirects through `www.redhat.com`.
+
+|`aws.amazon.com`
+|443
+|The `iam.amazonaws.com` and `sts.amazonaws.com` sites redirect through `aws.amazon.com`.
+
+|`catalog.redhat.com`
+|443
+|The `registry.access.redhat.com` and `http://registry.redhat.io` sites redirect through `catalog.redhat.com`.
```

Comment:
For catalog.redhat.com, the website address should be https:, not http:. Everything else looks great!